lib/function.g

```diff
@@ -566,8 +565,12 @@ InstallMethod( ViewObj, "for a function", true, [IsFunction], 0,
     nams := NAMS_FUNC(func);
     narg := NARG_FUNC(func);
     if narg < 0 then
+        isvarg := true;
         narg := -narg;
     fi;
+    if narg = 1 and nams[1] = "arg" then
```

Comment:
Turns out this check can fail: Apparently is possible for nams to be `fail`, when looking at C level code. The resulting function, when printed, is shown like this:
```
function ( <<arg-1>> )
    <<kernel or compiled code>>
end
```
A reproducible (but not yet minimal) test case:
```
METHODS_OPERATION(Socle,1);
```